recOrder/plugin/main_widget.py

```diff
@@ -56,6 +56,26 @@ class MainWidget(QWidget):
     # Initialize Custom Signals
     log_changed = Signal(str)
 
+    # Initialize class attributes
+    disabled_button_style = "border: 1px solid rgb(65,72,81);"
+    bf_keywords = [
+        "bf",
+        "brightfield",
+        "bright",
+        "labelfree",
+        "label-free",
+        "lf",
+        "label",
+        "phase",
+        "ph",
+    ]
+    msg = (
```

Comment:
Please consider using a more descriptive name for this variable since it's now a class attribute that can be accessed through `MainWidget.msg`.

Reply:
I might be missing some constraints here but this should work so that `msg` is not in the namespace:
```py
# delete the `msg = ...` line
no_bf_msg = textwrap.wrap('No ... keywords: '.join(bf_keywords), width=70)
```

Reply:
Thanks for pointing this out...still learning the ropes of these class attributes. Fixed.